src/common/options.cc

```diff
@@ -4948,6 +4948,9 @@ std::vector<Option> get_global_options() {
     .set_description("Enforces specific hw profile settings")
     .set_long_description("'hdd' enforces settings intended for BlueStore above a rotational drive. 'ssd' enforces settings intended for BlueStore above a solid drive. 'default' - using settings for the actual hardware."),
 
+    Option("bluestore_ioring", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
```

Comment:
@markhpc we don't have an experimental flag anymore. but this setting is marked with "Option::LEVEL_ADVANCED", and is disabled by default. i think it's good enough. or do you have a specific idea how to mark it experimental?